source/index.html

```diff
@@ -48,8 +52,12 @@ <h1 class="selection__title">
         <p class="selection__description">
           Занялся собой? Займись котом!
         </p>
-        <img class="selection__image" src="./img/index-can-mobile.png" width="280" height="270"
-          alt="Пример питания для котов Кэт Энерджи Про с курицей 500 грамм">
+        <picture>
+          <source media="(min-width:1300px)" srcset="./img/index-can-desktop.png, /img/index-can-desktop@2.png 2x">
+          <source media="(min-width:768px)" srcset="./img/index-can-tablet.png, /img/index-can-tablet@2.png 2x">
```

Comment:
Куда потерялся `x`?
Смотри вообще через девтулз картинки на 3х разрешениях и в двух вариантах плотности
```suggestion
          <source media="(min-width:1300px)" srcset="./img/index-can-desktop.png, /img/index-can-desktop@2x.png 2x">
          <source media="(min-width:768px)" srcset="./img/index-can-tablet.png, /img/index-can-tablet@2x.png 2x">
```